helpers/make_network_dirs_main.py

```diff
@@ -0,0 +1,118 @@
+"""Script that creates all directories"""
+import os
+import sys
+sys.path.append("D:/research-and-development")
+from src.utils.helpers import tree_to_list
+from src.utils.local_file_mods import rd_file_exists
+
+# Change to the project repository location
+my_wd = os.getcwd()
+my_repo = "R:\\"
+if not my_wd.endswith(my_repo):
+    os.chdir(my_repo)
+
+
+def make_dir(dir):
+    """Creates a directory bassed on the provided folder_path."""
+    # create directory if it does not exist
+
+    if rd_file_exists(dir, raise_error=False) is False:
+        os.makedirs(dir)
+        print(f"Created {dir}")
+    else:
+        print(f"Directory {dir} already exists.")
+
+
+def run_make_dirs():
+    """Creates all directories based on the `tree` structure.
+
+    tree (dict) converted to list of lists by function src.utils.helpers.tree_to_list.
+    make_dir function is called for item in the list of directories. Itertates through
+    years (list) and surveys (list) to create the directories for said years and
+    surveys.
+
+    """
+
+    root = "BERD Results System Development 2023/DAP_emulation"
+
+    years = ["2022", "2023"]
+    surveys = ["BERD", "PNP",]
+
+    for year in years:
+        for survey in surveys:
+
+            tree = {
+                f"{year}_surveys": {
```

Comment:
line 45 can be outside the loop so it isn't done each time.

Reply:
Implementing this suggestion would actually add complexity to the script. One would have to 1. check if the folder are already created 2. create an `if` `else` statement to create or not create a new folder dependant on if the folder does/doesn't exist. Considering this script is not part of the pipeline, and will only be used once in a blue moon, the 
current genuine inefficiency has zero impact. I suggest we keep the current iteration.